completions/chflags

```diff
@@ -0,0 +1,31 @@
+# chflags(1) completion                                      -*- shell-script -*-
+
+_comp_cmd_chflags()
+{
+    local cur prev words cword comp_args
+    _comp_initialize -- "$@" || return
+
+    if [[ $cur == -* ]]; then
+        # Complete -options
+        local w opts=""
+        for w in "${words[@]}"; do
+            [[ $w == -R ]] && opts="-H -L -P" && break
+        done
+        _comp_compgen -- -W '-f -h -v -R $opts'
```

Comment:
FreeBSD (15.0) has `-x`.
```suggestion
        _comp_compgen -- -W '-f -h -v -x -R $opts'
```

Reply:
If `-x` is not available in other BSDs, I think we should reference `$OSTYPE` for `-x` the same as for the first word.

Reply:
44f29dae2c11f6e228d7e947e9aa0e6d9c1fb106